packages/core/echo/echo-schema/src/testing/schema.ts

```diff
@@ -90,7 +90,7 @@ export class Contact extends TypedObject<Contact>({
     name: S.String,
     username: S.String,
     email: S.String,
-    tasks: S.suspend((): S.mutable<S.Array$<ref<Task>>> => S.mutable(S.Array(ref(Task)))),
+    tasks: S.suspend((): S.mutable<S.Array$<Ref$<Task>>> => S.mutable(S.Array(Ref(Task)))),
```

Comment:
what does suspend do?

Reply:
It's necessary to allow self-referential schema